docs/questions/query-builder/expressions/countif.md

```diff
@@ -0,0 +1,273 @@
+---
+title: CountIf
+---
+
+# CountIf
+
+`CountIf` counts the total (not unique) rows in a table that match a condition.
+
+Syntax: `CountIf(condition)`.
+
+Example: in the table below, `CountIf([Plan] = "Basic")` would return 3.
+
+| ID  | Plan        |
+|-----|-------------|
+| 1   | Basic       |
+| 2   | Basic       |
+| 3   | Basic       |
+| 4   | Business    |
+| 5   | Premium     |
+
+> [Aggregations](../expressions-list.md#aggregations) like `CountIf` should be added to the query builder's [**Summarize** menu](../../query-builder/introduction.md#summarizing-and-grouping-by) > **Custom Expression** (scroll down in the menu if needed).
+
+## Parameters
+
+`condition` is a [function](../expressions-list.md#functions) or [conditional statement](../expressions.md#conditional-operators) that returns a boolean value (`true` or `false`).
```

Comment:
This phrasing is strange. Makes it seem like the word `condition` is a function you can invoke (like `condition()`.